lib/util/StackedSetMap.js

```diff
@@ -31,11 +79,17 @@ class StackedSetMap {
 		}
 	}
 
+	/**
+	 * @param {K} item the item to test
+	 * @returns {boolean} true if the item exists in this set
+	 */
 	has(item) {
 		const topValue = this.map.get(item);
-		if (topValue !== undefined) return topValue !== TOMBSTONE;
+		if (topValue !== undefined) {
+			return topValue !== TOMBSTONE;
+		}
 		if (this.stack.length > 1) {
-			for (var i = this.stack.length - 2; i >= 0; i--) {
+			for (let i = this.stack.length - 2; i >= 0; i--) {
```

Comment:
why not `const`?

Reply:
Because it is a _traditional_ for-loop and `i` needs to be updated at each iteration 😉